profiler/snippets/snippets.go

```diff
@@ -28,7 +29,12 @@ func main() {
 		ServiceVersion: "1.0.0",
 		// ProjectID must be set if not running on GCP.
 		// ProjectID: "my-project",
-	}); err != nil {
+	},
+		// For OpenCensus user for Cloud Trace and Cloud Monitoring:
+		// This option prevents the Profiler agent to send Span data
+		// when the agent send profile data to the backend.
+		option.WithTelemetryDisabled(),
```

Comment:
It's a little hard for me to follow this `if` declaration with the `Start` call with the inline config and this option. Perhaps we should declare the config and opts separately?

```
cfg := profiler.Config{
	// ...
}
// For OpenCensus...
opts := []option.ClientOption{option.WithTelemetryDisabled()}
if err := profiler.Start(cfg, opts...); err != nil {
	// ...
}
```

Reply:
I agreed to that. I updated the lines for config and option to have separate declaration.